src/main/java/de/uni_tuebingen/ub/nppm/db/SaveHelper.java

```diff
@@ -82,7 +136,22 @@ public static void updateAttribute(String table, String attribute, String value,
         }
     }
 
+    public static void insertNewAttribute(String table, String attributeOne, String attributeTwo, String attributeThree, int valueOne, int valueTwo, String valueThree) throws Exception {
```

Comment:
For insert, i would recommend the same things as for the update.

It doesn't make much sense to hardcode an inserting function with a fixed number of parameters.
Instead, the function header should look like this:

`public static void insert(String table, Map<String,String> columnsAndValues) throws Exception`

And by the way: If these update/insert implementations are good enough, we could even think about moving them to the AbstractBase later (but maybe we should rather keep them protected then).

Reply:
done look at comment: insert function in saveHelper renewed